tests/Database/DatabaseConnectionFactoryTest.php

```diff
@@ -13,6 +13,7 @@
 
 class DatabaseConnectionFactoryTest extends TestCase
 {
+    /** @var DB */
```

Comment:
Remove this
```suggestion
```

Reply:
You want to add docblocks everywhere else, but here where it is really needed for completion, you want to remove it? I don't understand the logic. 🤔